locales/en-US/server.ftl

```diff
@@ -217,6 +219,18 @@ annotationColorSeaGreen =
 annotationColorGrey =
     .title = Grey
 
+# Note: annotationTextSize is a title for text size selection dropdown.
+annotationTextSize =
+    .title = Text Size
+# Values shown in text size selection dropdown
+textSizeSmall = Small
+textSizeMedium = Medium
+textSizeLarge = Large
+# Confirm and Cancel button title shown when using text tool
+annotationTextConfirmButton =
+    .title = Confirm Text
+annotationTextCancelButton =
+    .title = Cancel Text
```

Comment:
These two tooltips seem a bit off. Using simply "Confirm" and "Cancel"?

Reply:
+1

Reply:
Done